### PR TITLE
use rpm instead of build haproxy from source

### PR DESCRIPTION
**Pull request by tdawson**

### CODE REVIEW

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

**Comment by tdawson** [comment]
[test]

File: images/router/haproxy-base/Dockerfile [hunk 1]
```diff
@@ -6,18 +6,11 @@
...
-RUN yum install -y gcc make openssl-devel pcre-devel tar wget socat && \
-    wget http://www.haproxy.org/download/1.5/src/haproxy-1.5.10.tar.gz  && \
-    tar xvzf haproxy-1.5.10.tar.gz && \
-    groupadd haproxy && \
-    useradd -g haproxy haproxy && \
-    cd haproxy-1.5.10 && make TARGET=linux2628 CPU=generic USE_PCRE=1 USE_OPENSSL=1 USE_ZLIB=1 && make install && \
-    cd .. && rm -rf haproxy-1.5.10 && \
+RUN yum -y install https://tdawson.fedorapeople.org/openshift/haproxy-1.5.4-2.el7.x86_64.rpm && \
     mkdir -p /usr/bin && \
     mkdir -p /var/lib/containers/router/{certs,cacerts} && \
```
Comment (detiber) [inline images/router/haproxy-base/Dockerfile]: @tdawson This and line 19 look suspicious...  I wouldn't expect the default haproxy RPMs to have those directories.

**Comment by tdawson** [comment]
I think I found the problem.  We have hard coded the command to start haproxy to /usr/local/sbin/haproxy, this it will always fail.  I changed it to /usr/sbin/haproxy to go along with this rpm change.

**Comment by sdodson** [comment]
LGTM, @pweil- mind giving this a merge if you're ok with it? Looks like tests are passing.

**Comment by pweil-** [comment]
LGTM  [merge]

**Comment by brenton** [comment]
[merge]

**Pull request merged**